### DISCUSSION TITLE
4.0.0-beta.1 release notes copyediting / drafting help welcomed

### DISCUSSION THREAD

**Discussion by ckerr** [transmission/transmission#3273] [General]
I've put a draft up at https://github.com/transmission/transmission/pull/3272 but it is currently barely even qualifies as a draft. The most work has gone into the 'Thanks' section but it's incomplete.

Basically, since this release has been in the works for ~2 years there is a HUGE laundry list of changes to sort / collate and to translate into a coherent set of release notes. Ideally I'd like something like the VS Code release notes https://code.visualstudio.com/updates/ which are fantastic.

Also I definitely want to start giving contributors credit in the same way that's done in the VSC release notes, e.g. linking their GitHub accounts and citing them by name. This is all part of the larger goal of being more welcoming to contributors and having a healthier ecosystem.

I thought I'd raise the discussion here since I know there are a few contributors who have offered to work on writing assignments, e.g. @GaryElshaw who offered to work on a newsletter. If we wound up with 4.0.0-beta.1 release notes that looked like a VSC release's notes I think that would be real progress.

Also TBD: I'd like to find some way of giving credit to contributors whose value wasn't directly related to feature commits, e.g. in giving code reviews, or testing other peoples' changes, and so forth. I think the project has several very active people whose contributions aren't in the commit logs.

**Comment by GaryElshaw** [comment]
Can you please make me a contributor, setting some reviewers would be great.

**Comment by ckerr** [comment]
One way that I've done this in the past on other projects is to use a hackmd page for drafting markdown. It's less friction than going through the PR/CI dance for revisions to a text document.

So, just to suggest something to move the ball forward -- what would you think about moving the text of that news-4.0.0-beta.1.md to a hidden hackmd page so that we could work on it there? Does that sound like a good idea and/or does anyone have a better workflow?

**Reply by GaryElshaw** [reply]
That sounds like a great idea.

**Comment by ckerr** [comment]
Anyone who wants to contribute, please sign up for an account on hackmd.io and email me (at my-firstname @charleskerr.com) the email account that you used to sign up & I'll send you the URL for the release notes draft :green_heart:

**Reply by GaryElshaw** [reply]
Mail on its way.

**Reply by ile6695** [reply]
I guess spam filters ate my mail.

**Comment by ckerr** [comment]
@ile6695 sorry for not getting back to you.

Since I've been bad at responding to email, I went ahead and merged the draft notes into `main` so that revisions can be done via PR.
